src/targets/public/main.jsx

```diff
@@ -41,7 +42,21 @@ function init () {
   if (directdownload) {
     cozy.client.files.getDownloadLinkById(id).then(
       link => window.open(`${cozy.client._url}${link}`, '_SELF')
-    )
+    ).catch(e => {
+      cozy.bar.init({
+        appName: data.cozyAppName,
+        appEditor: data.cozyAppEditor,
+        iconPath: data.cozyIconPath,
+        lang: data.cozyLocale,
+        replaceTitleOnMobile: true
+      })
```

Comment:
We don't.
For shared files, we don't need the cozy-bar (at least not as long as we don't use the file viewer for shared files).
We may still init the cozy-bar before, but it will take time.

I think what we should do is to use a static html/css page for such error. But in that case, we won't have i18n, at least as long as we don't do something without JavaScript.
Maybe the backend could send us such a page.